### ISSUE TITLE
Static polygons in plotly? Is it possible?

### ISSUE THREAD

**Issue by tpbilton** [ropensci/plotly#1251]
Hi, I have been trying to create a plot where I have some points that I want to be interactive with hover-info etc but I want to include two polygon areas without any of the interactive stuff.

For MWE:
```
library(plotly)
data("iris")
xsq <- function(x) sqrt(x)
x <- c( c(seq(0,10,0.001),0,0), c(0.5,10,10,0.5))
y <- c( c(xsq(x[1:length(seq(0,10,0.001))]),xsq(max(x)),0), c(0,2,0,0))

## produce the plotly plot
plot_ly(x = x, y = y, alpha = 0.1, opacity=0.1) %>%
  add_polygons(hoverinfo = "none", color = I("red"),showlegend=F) %>%
  add_polygons(x = c(0.5,10,10,0.5), y=c(0,2,0,0), hoverinfo = "none", 
               color = I("blue"), showlegend=F) %>%
  add_markers(x=iris$Sepal.Length,y=iris$Sepal.Width-2,  opacity=1, alpha=1,
              color=iris$Species, hoverinfo="text", text=iris$Species)
```

This is sorting of giving me what I want but I a couple of problems:
1. The hover informative for the points in the polygon regions are not appearing
2. If I select a group, then the two polygons also disappear from view. I would like to keep the polygon present at all times even when only a single groups of points is to be selected

I was also trying to use ggplot but wasn't having any luck there.

Basically what I think I want is a way to add two static polygons to a plotly plot. Does anyone have any other suggestions/ideas?

Thanks,
Timothy

**Comment by cpsievert** [comment]
Thanks!

As per our [contributing guidelines](https://github.com/plotly/plotly.js/blob/master/CONTRIBUTING.md), github issues are reserved for bug reports and feature requests only. So I'll have to close your ticket.

Please use our community forums at [community.plot.ly](http://community.plot.ly/) for posts of the likes.
